03/app.js

```diff
@@ -6,26 +6,57 @@ class Article extends React.Component {
         comments: [],
     }
     
+
+    submitHandler = (e) =>{
+        e.preventDefault();
+        this.addComment(e);
+    }
+
+    addComment(e) {
+        const {content: userComment} = e.target.elements;
+    
+        this.setState({
+            comments: [...this.state.comments, userComment.value]
+        });
+
+        this.clearFileds(e);
+    }
+
+    clearFileds(e) {
+        return e.target.reset();
+    }
+
+    showComment() {
+        const {comments} = this.state;
+        const commentJSX = comments.map(item => {
+            return <li>{item}</li>
+        });
+
+        return commentJSX;
+    }
+    
     render() {
         const {title, body} = this.props;
         return (
             <article>
                 <h1>{ title }</h1>
                 <p>{ body }</p>
                 <section>
-                    <form>
+                    <form onSubmit={this.submitHandler }>
                         <div>
                             <label>
                                 <textarea 
```

Comment:
Ten formularz nie jest kontrolowany.
Jego zawartość nie zależy od `state` - powinieneś to zmienić.
Ta forma jest preferowana w React.